exporter/datadogexporter/factory_test.go

```diff
@@ -222,6 +234,12 @@ func TestLoadConfig(t *testing.T) {
 			},
 			IgnoreResources: []string{},
 		},
+		Logs: LogsConfig{
+			TCPAddr: confignet.TCPAddr{
+				Endpoint: "https://http-intake.logs.datadoghq.test",
+			},
+			SendLogRecordBody: true,
+		},
 		HostMetadata: HostMetadataConfig{
```

Comment:
I prefer this style, based on our coding conventions, where there are no empty lines between these blocks. Can we make it consistent throughout?